CMakeLists.txt

```diff
@@ -150,6 +150,10 @@ if (VIAMCPPSDK_ENFORCE_COMPILER_MINIMA)
     if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 10.0)
       message(FATAL_ERROR "Insufficient Apple clang version: XCode 10.0+ required")
     endif()
+  elseif (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Comment:
This establishes a compiler minimum for Windows. Since there is no history here, the newest Visual Studio seems like fair game. We can always try to downgrade it later if we find a need.